code/modules/surgery/bodyparts/_bodyparts.dm

```diff
@@ -1134,3 +1134,27 @@
 		owner.update_body_parts()
 	else
 		update_icon_dropped()
+
+/obj/item/bodypart/emp_act(severity)
+	. = ..()
+	if(. & EMP_PROTECT_WIRES || !(bodytype & BODYTYPE_ROBOTIC))
+		return FALSE
+	owner.visible_message(span_danger("[owner]'s [src.name] seems to malfunction!"))
+
+	var/time_needed = 4 SECONDS
+	var/brute_damage = 2.5
+	var/burn_damage = 7.5
+
+	if(severity == EMP_HEAVY)
+		time_needed *= 2
+		brute_damage *= 2
+		burn_damage *= 2
```

Comment:
under this system, a full-augged person will take ~55 effective burn damage from a heavy emp (if I did my math right).

is this your intention?

Reply:
I don't have an intention. Better damage from EMPs than 20 second stuns though.

Reply:
Gonna go ahead and reduce the dmg a bit since 20 second stuns are demanded IG